setup.py

```diff
@@ -17,6 +17,20 @@
 
 import setuptools
 
+# Disable version normalization performed by setuptools.setup()
```

Comment:
Is this intentionally being rolled into this PR?

Reply:
This line is already in master. I think the branch is out of date.

Reply:
I'm going to retry the migration.

Reply:
This issue was due to https://github.com/googleapis/repo-automation-bots/issues/1752. I'm going to force push the branch.